src/main/kotlin/com/openlattice/organizations/ExternalDatabaseManagementService.kt

```diff
@@ -124,14 +124,20 @@ class ExternalDatabaseManagementService(
         return organizations.keys
     }
 
-    fun getExternalDatabaseTables(orgId: UUID): Set<OrganizationExternalDatabaseTable> {
-        return organizationExternalDatabaseTables.values(belongsToOrganization(orgId)).toSet()
+    fun getExternalDatabaseTables(orgId: UUID): Map<UUID, OrganizationExternalDatabaseTable> {
+        return organizationExternalDatabaseTables.entrySet(belongsToOrganization(orgId))
```

Comment:
seems like you should be able to just `return organizationExternalDatabaseTables.entrySet(belongsToOrganization(orgId))`

Reply:
that returns a set<map.entry<uuid, table>>, which is fine too. but I will probably still have to do a map{} in the controller.